nncf/experimental/torch/nas/bootstrapNAS/training/progressive_shrinking_builder.py

```diff
@@ -88,15 +91,27 @@ def _get_algo_specific_config_section(self) -> Dict:
     def _build_controller(self, model: NNCFNetwork) -> "ProgressiveShrinkingController":
         elasticity_ctrl = self._elasticity_builder.build_controller(model)
         schedule_params = NASSchedulerParams.from_config(self._algo_config.get("schedule", {}))
+        compression_loss_func = self._build_compression_loss_function(model)
         return ProgressiveShrinkingController(
             model,
             elasticity_ctrl,
             self._bn_adaptation,
             self._progressivity_of_elasticity,
             schedule_params,
             self._lr_schedule_config,
+            compression_loss_func,
         )
 
+    def _build_compression_loss_function(self, model: NNCFNetwork) -> "PTCompressionLoss":
+        compression_builder = create_compression_algorithm_builder(self._algo_config)
```

Comment:
Could you please verify what will happen if there's no KD algorithm section in the config? Will it return `ZeroCompressionLoss`?

Reply:
Yes. When nncf config doesn't include KD algorithm section, this function will return `ZeroCompressionLoss`. When KD algorithm is included, then this function will return `KnowledgeDistillationLoss`.